peekaboo/toolbox/cortex.py

```diff
@@ -57,6 +57,17 @@ def __init__(self, analyzer):
 class CortexAnalyzerReport:
     """ Cortex analyzer report base class. """
 
+    @classmethod
+    def get_element_from_list_of_dicts(cls, dictionary, ident_key, ident_value, default = {}):
+        """ Search a list of dicts for an element with a matching value for an
+            identifying key """
+        element = [submodule for submodule in dictionary
```

Comment:
Looking at this (again), I wonder if this shouldn't rather read `dict for dict in list`?

Reply:
You are right of course. I fixed that.
And added unittests.
I have force-pushed to your branch again 🙈 We should stop this it gives me pain